src/common/base/Status.h

```diff
@@ -148,6 +149,7 @@ class Status final {
         kSyntaxError      = 201,
         kStatementEmpty   = 202,
         kMalformedRequest = 203,
+        kPartiallyFailed  = 204,
```

Comment:
I don't think it's good to add error code, because there are other users who want to be successful in this situation, so you can add a `gflag` and let the users set whether to return success or failure in this situation.

Reply:
who wants the partial success as default? AFAIK, none.

Reply:
> who wants the partial success as default? AFAIK, none.

We need to make sure that the user all think this, and why we return succeeded at the beginning.

Reply:
Yeah, as a storage, it is better to strictly prevent data loss. Most businesses cannot tolerate data loss, especially as a default behavior.
Indeed, we can do better here, because a few businesses may think that high availability is more important than data consistency.
So here we can consider two steps for improvement.

The first step, like this PR, return an error code by default when partial failure.

The second step is to consider whether to accept the proposal in this issue: https://github.com/vesoft-inc/nebula/issues/2304, such as allowing users to pass in class RequestConf when creating GraphClient to customize how to deal with partial failure.

Reply:
Agreed with return error code if succeeded partially.